CenterForOpenScience/ember-osf-web
# [ENG-4161][ENG-4165] Add node metadata to projects and registrations

Tickets: 
- [ENG-4161]
- [ENG-4165]

## Purpose

Allow users on nodes and registrations to view their metadata. Note that tests and styling are not in yet, and will be added in a future PR.

## Summary of Changes

1. Fix up the manager
2. Add translation strings
3. Create a component for the node metadata form
4. Add component to guid-node/metadata
5. Add component to registration's metadata
6. Fix metadata factories

## Screenshot(s)

<img width="1322" alt="Screen Shot 2022-11-18 at 10 38 21 AM" src="https://user-images.githubusercontent.com/6599111/202798374-60e4d264-7fef-4085-b8e4-1e024c7da9ba.png">
<img width="1168" alt="Screen Shot 2022-11-18 at 10 38 00 AM" src="https://user-images.githubusercontent.com/6599111/202798375-ebacf5d7-e4da-44de-9e48-aa6bc3a16c52.png">


## Side Effects

I don't believe so.

## QA Notes

These changes should be isolated to the project's metadata page and the registration's metadata page.


[ENG-4161]: https://openscience.atlassian.net/browse/ENG-4161?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[ENG-4165]: https://openscience.atlassian.net/browse/ENG-4165?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
